src/machines/modelingMachine.ts

```diff
@@ -1819,6 +1763,82 @@ export const modelingMachine = setup({
         }
       )
     }),
+    revolveAstMod: fromPromise<
```

Comment:
@nadr0 @franknoirot Largely the same code as above, just moved to the fromPromise style, adding the nodeToEdit block and updateModelingState for consistency with other commands. This still has a lot of the get and set ag logic in the addRevolve.ts file.